spec/factories/comments_factory.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :comment do
+    title "MyString"
+    body "MyString"
+    Message nil
```

Comment:
to clarify, the syntax is `attribute value`, so a Comment has a Message as an attribute `message`, for relations like this factorygirl is smart enough to go to the message factory when it sees just `message` without a value as the attribute. it's like writing `message { FactoryGirl.create :message }`

this was probably generated though, right?